adapters/handlers/rest/configure_server.go

```diff
@@ -36,7 +36,7 @@ import (
 // This function can be called multiple times, depending on the number of serving schemes.
 // scheme value will be set accordingly: "http", "https" or "unix"
 //
-// we will set it through configureAPI() as it needs access to resources that
+// we will set it through ConfigureAPI() as it needs access to resources that
```

Comment:
can we change it back to `configureAPI` as we are still using this method for initing Weaviate